app/additional-data/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import { Container, Typography, Button, Stack, Box, Fade } from '@mui/material';
+import { QuestionCircle } from 'react-bootstrap-icons';
+import { useFrameworkInstanceStore } from '@/store/selected-framework-instance';
+import Tip from '@/components/Tip';
+import { AdditionalDatasheetEditor } from '@/components/AdditionalDatasheetEditor';
+
+export default function AdditionalDataPage() {
+  const baselineYear = useFrameworkInstanceStore((state) => state.baselineYear);
+
+  const handleButtonClick = () => {
+    console.log('The Modal!');
+  };
+
+  return (
+    <Fade in>
+      <Container>
+        <Stack direction="row" alignItems="baseline" spacing={1}>
+          <Typography variant="h3">Additional Historical Data</Typography>
```

Comment:
Let's make the element a h1 (while still keeping the h3 styles) for accessibility

```suggestion
          <Typography component="h1" variant="h3">Additional Historical Data</Typography>
```